src/pybind/mgr/nfs/export.py

```diff
@@ -97,6 +105,22 @@ def normalize_path(path: str) -> str:
     return path
 
 
+class ExportClient(CephfsClient):
```

Comment:
I don't understand - why not have `ExportMgr` class subclass CephfsClient?

Reply:
I did try it but jenkins make check still complains

```
Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 400, in create_export
    return self.create_cephfs_export(**kwargs)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 643, in create_cephfs_export
    with open_filesystem(self, fs_name) as fs_handle:
  File "/usr/lib/python3.10/contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/mgr_util.py", line 351, in open_filesystem
    if fsc.is_stopping():
AttributeError: 'ExportMgr' object has no attribute 'is_stopping'
```

I'll try re-run it

Reply:
Can you share the code diff?

Reply:
It's the first commit in the PR

Reply:
this has been fixed by creating instance of `CephfsClient` in `__init__`